CENSUS_METADATA.md

```diff
@@ -0,0 +1,778 @@
+`Dataset_Variable`. Zero or more geographic variables can be specified for a single table in `Dataset_Variable.csv`.
+`vars` will contain at most one geographic variable. This will be the first element on the list and will
+be the geographic variable in `Dataset_Variable.csv` which has `Lowest_Geog_Variable_Flag` set to `Y`.
+All other geographic variables are listed in `TableMetadata.Alternate_Geographic_Variables`.
+
+## TableMetadata
+
+All the fields in `TableMetadata` are user defined. It contains information from `Dataset.csv` that
+is not included in the `Table` object.
+
+The object also contains data from `Related_Dataset.csv`, `Dataset_Keyword.csv`, `Publication_Dataset.csv`
+and `Release_Dataset.csv`.
+
+The `Signed_Off_Flag` and `Id` field in `Classification.csv` are ignored.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `Dataset_Mnemonic_2011` | `String` | `Dataset.Dataset_Mnemonic_2011` | |
+| `Geographic_Coverage` | `String!` | `Dataset.Geographic_Coverage` | `Dataset.Geographic_Coverage_Welsh` |
+| `Dataset_Population` | `String!` | `Dataset.Dataset_Population` | `Dataset.Population_Welsh` |
+| `Last_Updated` | `String` | `Dataset.Last_Updated` | |
+| `Unique_Url` | `String` | `Dataset.Unique_Url` | |
+| `Contact` | `Contact` | Keyed on `Dataset.Contact_Id` | |
+| `Version` | `String!` | `Dataset.Version` | |
+| `Related_Datasets` | `[String]!` | List of `Related_Dataset.Related_Dataset_Mnemonic` values keyed on `Related_Dataset.Dataset_Mnemonic` | |
+| `Keywords` | `[String]!` | List of `Dataset_Keyword.Dataset_Keyword` values keyed on `Dataset_Keyword.Dataset_Mnemonic` | List of `Dataset_Keyword.Dataset_Keyword_Welsh` values keyed on `Dataset_Keyword.Dataset_Mnemonic` |
+| `Publications` | `[Publication]!` | Keyed on `Publication_Dataset.Dataset_Mnemonic` | |
+| `Census_Releases` | `[Census_Release]!` | Keyed on `Release_Dataset.Dataset_Mnemonic`/`Census_Release_Number`| |
+| `Statistical_Unit` | `Statistical_Unit!` | Keyed on `Dataset.Statistical_Unit` | |
+| `Alternate_Geographic_Variables` | `[String!]` | List of alternate geographic variable names which are available for this table sourced from `Dataset_Variables.csv` | |
+
+## Dataset
+
+The `Dataset` object is a built-in type in `cantabular-metadata`. The `name`, `label` and `description`
+fields are sourced from `Database.csv`. The other fields in the file are contained in the `DatasetMetadata`
+object.
+
+| Field | GraphQL Type | Source (en) | Source (cy) |
+| --- | --- | --- | --- |
+| `name` | `String!` | `Database.Database_Mnemonic` | |
+| `label` | `String` | `Database.Database_Title` | `Database.Database_Title_Welsh` |
+| `description` | `String` | `Database.Description` | `Database.Database_Description_Welsh` |
+| `meta` | `DatasetMetadata!` | Additional fields from `Dataset.csv` | |
+| `vars(names: [String!]!)` | `[Variable]!` | | |
```

Comment:
Is the `(name: [String!]!)` bit intentional?

Reply:
I don't think so.